README.md

```diff
@@ -1,12 +1,13 @@
-# Elevator Call API example code project
+# KONE APIs example code project
 
 ## Introduction
 
 The example codes on this document will introduce the main concepts of KONE APIs. By setting up and running this example code project, you can apprehend the main idea and flow of using KONE APIs. It is highly recommended to read [the technical API documentation](https://dev.kone.com/api-portal/dashboard/developer-guide/overview-api) before starting to use this example project.
 
 - With **Authentication API** you can receive the access token that enables you to make an API call
 - With **Building API** you can retrieve information about your building, and
-- With **Elevator Call WebSocket API** you can execute elevator calls and receive real-time data about the call and the assigned elevators.
+- With **Elevator Call WebSocket API** you can execute elevator calls and receive real-time data about the call and the assigned elevators, and
+- With **Operational API** you can retrieve information about your accessible equipment, its maintenance status, list of service orders for the equipment and their details.
```

Comment:
Operational APIs in plural as Service Info and Equipment status  are separated everywhere.